pageserver/src/bin/pageserver.rs

```diff
@@ -228,14 +231,14 @@ fn main() -> Result<()> {
                 .long("listen-pg")
                 .alias("listen") // keep some compatibility
                 .takes_value(true)
-                .help("listen for incoming page requests on ip:port (default: 127.0.0.1:5430)"),
+                .help(formatcp!("listen for incoming page requests on ip:port (default: {DEFAULT_PG_LISTEN_ADDR})")),
```

Comment:
Note that standard `format!` does not have [implicit arguments](https://github.com/rust-lang/rfcs/blob/master/text/2795-format-args-implicit-identifiers.md) yet — suggested in `const_format`'s docs.